raiden/ui/cli.py

```diff
@@ -275,12 +275,12 @@ def run(ctx, **kwargs):
 
         app_ = ctx.invoke(app, **kwargs)
 
-        app_.discovery.register(
+#         spawn address registration to avoid block while waiting for the next block
```

Comment:
As I said in gitter, just push the new commit in master for now. But on another note, for so small fixes please do not make new commits, but simply amend the previous commit and force push.

Sme things goes for rebasing. Force pushing may sound bad, but it's perfectly fine if and only if it's on a feature branch only you work on.